include/SFML/System/Vector2.hpp

```diff
@@ -88,77 +87,77 @@ class Vector2
     /// * Vector2(r, phi) == Vector2(r, phi + n * 360_deg)
     ///
     ////////////////////////////////////////////////////////////
-    Vector2(T r, Angle phi);
-    
+    SFML_SYSTEM_API Vector2(T r, Angle phi);
```

Comment:
I now realize `<i><b>(floating-point)</b></i>` is missing from this method documentation.

Reply:
> Isn't it also required to declare explicit template instantiations in header files?
> 
> Something like this:
> 
> ```c++
> extern template class sf::Vector2<float>;
> ```

I don't think it would really help in this case, because only a partial subset of the template member functions are explicitly instantiated. Where would you put the `extern template` declaration?

Reply:
> I now realize `<i><b>(floating-point)</b></i>` is missing from this method documentation.

Will fix